app/policies/group_policy.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+# Class for Group permissions
+class GroupPolicy < ApplicationPolicy
+  def show?
+    true
+  end
+
+  def index?
```

Comment:
yeah, I think that the index actions will end up being more complicated so they belong in separate issues.